install.sh

```diff
@@ -297,6 +297,49 @@ ops_link() {
   fi
 }
 
+checkHWAccelSupport() {
+  echo ""
+
+  local hwSupported=false
+  local haveRights=false
+
+  if [ "$OS" = "linux" ]; then
+    local group="kvm"
+    local username=`whoami`
+    local hw_support="grep -woE 'svm|vmx' /proc/cpuinfo | uniq"
```

Comment:
The shell command must be enclosed in ``, not "".

Reply:
Thanks, changed